README.md

```diff
@@ -18,6 +18,17 @@ npm install graphql-constraint-directive@v2
 ```
 
 ## Usage
+
+### Schema wrapper
+
+Implementation based on schema wrappers - basic scalars are wrapped as custom scalars with validations. 
+
+Pros:
```

Comment:
```suggestion
#### Benefits
```

Can we rename these to Benefits/Caveats instead of pros/cons?

Reply:
Sure thing